tests/bud.bats

```diff
@@ -39,6 +39,39 @@ _EOF
   validate_instance_compression "3" "$list" "arm64" "zstd"
 }
 
+@test "bud: build manifest list and --add-compression with containers.conf" {
+  local contextdir=${TEST_SCRATCH_DIR}/bud/platform
+  mkdir -p $contextdir
+
+  cat > $contextdir/Dockerfile1 << _EOF
+FROM alpine
+_EOF
+
+  cat > $contextdir/containers.conf << _EOF
+[engine]
+add_compression = ["zstd"]
+_EOF
```

Comment:
These can be regular files in the source tree, since their content isn't dynamic.